src/main/scala/org/allenai/deep_qa/experiments/SciQDatasetExperiments.scala

```diff
@@ -0,0 +1,302 @@
+package org.allenai.deep_qa.experiments
+
+import org.allenai.deep_qa.experiments.datasets.SciQDatasets
+import org.allenai.deep_qa.experiments.datasets.ScienceDatasets
+import org.allenai.deep_qa.pipeline.Evaluator
+
+import com.mattg.util.FileUtil
+import org.json4s.JsonDSL._
+import org.json4s._
+
+object SciQDatasetExperiments {
+  val fileUtil = new FileUtil
+
+  def experiment(
```

Comment:
It'd be nice to have this be in some general place somewhere, instead of duplicated like this.  Not sure where to put it, though.